contracts/token/Token.sol

```diff
@@ -69,8 +69,6 @@ contract Token is IToken, AgentRole {
     *  @param _name the name of the token
     *  @param _symbol the symbol of the token
     *  @param _decimals the decimals of the token
-    *  @param _version the version of the token, current version is 3.0
-    *  @param _onchainID the address of the onchainID of the token
```

Comment:
why did you remove this?

Reply:
_onchainID is still a parameter

Reply:
> _onchainID is still a parameter

Didn't notice that I have deleted this, fixed :)